app/(product)/public-cloud/(product)/product/[licencePlate]/@decision/page.tsx

```diff
@@ -67,6 +64,12 @@ export default function RequestDecision({ params }: { params: { licencePlate: st
     }
   }, [data]);
 
+  useEffect(() => {
+    if (!session?.user?.roles?.includes('admin')) {
```

Comment:
We we use `session?.isAdmin` as it is available now?

Reply:
Yes I will use that